src/crispy/logstore.h

```diff
@@ -68,164 +68,156 @@ class SourceLocation
     char const* _functionName;
 };
 
-// #if __has_include(<source_location>) && !defined(_WIN32)
-// using source_location = std::source_location;
-// #elif __has_include(<experimental/source_location>)
-// using source_location = std::experimental::source_location;
-// #else
-using source_location = SourceLocation;
-// #endif
```

Comment:
is this all really not needed anymore? hmm... our own implementation of `source_location` should be only used on systems that do not provide `std::source_location` (which is available in C++20).

Reply:
it was commented so do we need then to create separate issue for this and then make new PR on top?

Reply:
ah right. I actually forgot why it was commented out. probably only because I wanted to test the own implementation and forgot to re-enable it, lol, my bad. I have no strong feelings here. Except that I think it's good to use from APIs provided by the stdlib, if available. Soonish we can raise the minimum required C++ version to C++20 anyways.

Reply:
> do we need then to create separate issue for this and then make new PR on top

I don't mind either way. it's probably simplest to just re-enable (in its own commit) in this PR. :)